qml/js/yleApi.js

```diff
@@ -73,6 +73,7 @@ var mapPrograms = function(programs) {
         return {
             id: program.id,
             title: program.title && program.title.fi,

```

Comment:
at some point we need to make changes to this in order to support localization properly for the API content also. Just a side note :)

Reply:
Yes that should be done at some point.